message_converter/src/converter_node.cpp

```diff
@@ -147,26 +93,36 @@ void SwiftNavRover::posLLHCallback(const libsbp_ros_msgs::MsgPosLlh::ConstPtr &
     int h_accuracy = msg->h_accuracy;
     int v_accuracy = msg->v_accuracy;
     int tow = msg->tow;
+    ros::Time t = lrm::convertGpsTimeToUtcRosTime(SwiftNavRover::week_, tow, SwiftNavRover::residual_, SwiftNavRover::leap_seconds_);
     int fixed_mode = (msg->flags) & (0b00000111);
-    boost::array<double, 9> covariance = {{h_accuracy, 0, 0, 0, h_accuracy, 0, 0, 0, v_accuracy}};
-    ros::Time t = ros::Time::now();
+    SwiftNavRover::positioning_mode_ = fixed_mode;
+    boost::array<double, 9> covariance = {{h_accuracy/1000.00, 0, 0, 0, h_accuracy/1000.00, 0, 0, 0, v_accuracy/1000.00}};
     SwiftNavRover::geodetic2ned(lat, lon, height, &n, &e, &d);
-    if(SwiftNavRover::satelliteCheck == false)
+    if(SwiftNavRover::satellite_check_ == false)
     {
-        // ROS_INFO_STREAM("Publishing lat-lon converted position");
+        ROS_INFO_STREAM("Publishing lat-lon converted position");
```

Comment:
this should be throttled to 1Hz, since this can publish at 10 Hz!

Reply:
Removing this in https://github.com/Humatics/ethz_piksi_ros/pull/1/commits/48bd5c2d7ede1fbed1f91ed38d857407147ea5e8, since this information will be conveyed by the `/rtk_available` topic